docs/src/development.md

```diff
@@ -228,6 +228,62 @@ corresponding Vim plugin
 mode [julia-emacs](https://github.com/JuliaEditorSupport/julia-emacs).
 
 
+## Debugging
+Julia offers several options for debugging. A classical debugger is available with the 
+[Debugger.jl](https://github.com/JuliaDebug/Debugger.jl) package or in the 
+[Julia extension for VS Code](https://www.julia-vscode.org/docs/stable/userguide/debugging/). 
+However, it can be quite slow and, at the time of writing (January 2023), does currently not work 
+properly with Trixi. The [Infiltrator.jl](https://github.com/JuliaDebug/Infiltrator.jl) package on 
```

Comment:
```suggestion
However, it can be quite slow and, at the time of writing (January 2023), currently does not work 
properly with Trixi. The [Infiltrator.jl](https://github.com/JuliaDebug/Infiltrator.jl) package on 
```